platformio_override.sample.ini

```diff
@@ -533,6 +533,7 @@ monitor_filters = esp32_exception_decoder
 # 433MHz RF remote example for esp32dev
 [env:esp32dev_usermod_RF433]
 extends = env:esp32dev
-build_flags = ${env:esp32dev.build_flags} -D USERMOD_RF433
+build_flags = ${env:esp32dev.build_flags}
```

Comment:
This is actually redundant now and would work equally well if deleted. The `${env:esp32dev.build_flags}` is however vitally important if the user wants to add WLED build flags that configure the usermod, so the example should still stay.

But maybe change it to this:

```ini
;; Uncomment and append your own build flags here if you want to configure the usermod:
; build_flags = ${env:esp32dev.build_flags}
``